### ISSUE TITLE
Featured Items

### ISSUE THREAD

**Issue by BigActual** [automatedlogistics/Theme2016#39]
Switch featured items from each one having an image fully stretched to alternating between that and the next one having a solid color background. Format like this; http://new-site.marismith.com/

**Comment by brashrebel** [comment]
Just curious: why?

**Comment by BigActual** [comment]
Clarity, usability, enhanced visual appeal, accessibility...

**Comment by brashrebel** [comment]
AKA, you think it looks better. Ha ha.

**Comment by BigActual** [comment]
AKA, Kevin and I.

**Comment by BigActual** [comment]
To clarify, format to match spatially with the other site. not branding. Maintain the ALS brand.

**Comment by d4mation** [comment]
The featured items being the rows at the bottom created in #17?

I've done a quick mock using ALS Blue as the background color for every other row. 

![image](https://cloud.githubusercontent.com/assets/7770631/15024151/b54d0cf0-1201-11e6-9290-edab11443e5a.png)

On the example site, she seems to have lots of different types of content within the solid background areas (Like that four square area that flips on hover and that carousel that takes over your mouse scrolling when you get to it on the page). We've got the same type of content for all of them, so it doesn't seem to work very well.

Maybe for the solid background areas we center all the text?

![image](https://cloud.githubusercontent.com/assets/7770631/15024314/60fad4c4-1202-11e6-8cdc-c89421276e69.png)

**Comment by BigActual** [comment]
You'll notice that the People of importance are on the left side of the images you've left stretched. Flip flop the buttons on those to the right side. For the sections with blue background and Yellow CTA text, switch to White background and change the yellow COPY to blue. In those items then, align the CTA and button on the left, not centered, but definitely keep the copy center justified in it's column. Place a smaller version of the image you've removed, and that used to be stretched, in the area to the right that's left behind. Make these new images the same size as the ones in the "red line" section above.

**Comment by d4mation** [comment]
Ah, the bit about the People of Importance being overlapped was due to my local environment being slightly out-of-sync with the test server. I've gone ahead and corrected that&mdash;bit overdue anyway.

I've gone ahead and implemented this locally. The smaller versions of the images scale in to view on-scroll just as the text slides in from the side. They are set to center themselves vertically but the images that are currently assigned to those rows are meant to be full-width banner images which is why they seemed a bit scrunched. Images with a larger height in pixels would work better for the even rows.

![image](https://cloud.githubusercontent.com/assets/7770631/15026863/9b16ad5c-120e-11e6-8f69-80532e387f1b.png)

**Comment by BigActual** [comment]
Agreed. Taller images would work better. Please adjust to fit and let me know if you need new versions. Can do. Now, just flip flop everything. By that I mean odd rows will have copy and call to action right and even rows will have copy and call to action left with picture to the right.

**Comment by d4mation** [comment]
I was able to simply crop the images and it works much better now. I had to move the "Trust, Teamwork and Personal Touch" CTA one row up since all Even rows now have the CTA text on the left of the row.

![image](https://cloud.githubusercontent.com/assets/7770631/15027745/3c4c6c62-1213-11e6-9f12-bad7b5f0925d.png)

**Comment by d4mation** [comment]
Just committed the changes in the above Screenshot to the test site and uploaded the cropped versions of the photos for the appropriate rows.

**Comment by BigActual** [comment]
You were right. Go with the Blue Background instead of white. However, use white copy instead of yellow. I'm reserving yellow for actionable items like links and buttons. That's a universal design element both online and offline.

**Comment by BigActual** [comment]
Also, on wider screens the items in this section are pushing out to the whole width. We've restricted max width in several other sections. Let's do that here as well. Call me or Slack me to discuss details if this does not make sense.

**Comment by d4mation** [comment]
You mean that the text is being pushed entirely out of sight? I managed to do that just now by adjusting the font-size within my browser to "Large". I'm going to look into this more.

![image](https://cloud.githubusercontent.com/assets/7770631/15322469/18a0df80-1c0b-11e6-8fc4-c2470508bcd6.png)

**Comment by d4mation** [comment]
Managed to fix it for this situation. Turns out it didn't like the way I was vertically aligning the text. Thankfully there's more than one way to do that.

![image](https://cloud.githubusercontent.com/assets/7770631/15322967/9a5d018c-1c0d-11e6-8a61-a7079f6b088c.png)

![image](https://cloud.githubusercontent.com/assets/7770631/15322950/8fa6714c-1c0d-11e6-896a-cbe34d8b0c27.png)

**Comment by d4mation** [comment]
Did a little bit of tweaking and I think I've got this looking better on larger (15"+) monitors without sacrificing the current layout.

The text and images fit within a container, but the background colors (and background images, in the case of even rows) are full-width to help hide the container. On ultra-wide monitors this is good since otherwise there'd be a blue rectangle with empty white space to either side.

Here's a photograph of my monitor so that the proportions can be shown easily. A screenshot would show only the rendered pixels (which would look the same on any 1080p display). [I've got a 23", 1080p Monitor](http://www.amazon.com/Acer-H236HL-bid-23-Inch-Widescreen/dp/B00AZMLIDQ?ie=UTF8&psc=1&redirect=true&ref_=oh_aui_detailpage_o06_s00).

![image](https://cloud.githubusercontent.com/assets/7770631/15332757/e8e31c66-1c34-11e6-88a0-8801004f4e84.png)

The container matches up with the Timeline above this section.
